app/modules/annotations/parameters.py

```diff
@@ -115,5 +115,6 @@ def remove(cls, obj, field, value, state):
             if keyword is None:
                 return False
             obj.remove_keyword(keyword)
+            keyword.delete_if_unreferenced()
```

Comment:
Surely this should be done by the obj.remove_keyword call above rather than relying on this?

Reply:
i was torn -- do we ever need a way in code to remove a keyword from an annotation _without_ checking if it should be deleted?  not really certain -- the only thing i could think of was for supporting an op=**move** situation.  but we dont have that now.  :)   i guess we could make this part of `remove_keyword()` now and go back to this behavior if we need it.  (or some other solution like `remove_dont_check_delete()` or whatever.